watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/peracetic_acid_disinfection/peracetic_acid_case_study.yaml

```diff
@@ -0,0 +1,49 @@
+base_currency: USD_2020
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.08
+    units: USD_2020/kWh
+  disinfection_solution:
+    value: 5.0
+    units: USD_2020/gallon
+global_parameters:
+  plant_lifetime:
+    value: 30
+    units: year
+  utilization_factor:
+    value: 0.85
+    units: dimensionless
+  land_cost_percent_FCI:
+    value: 0
+    units: dimensionless
+  working_capital_percent_FCI:
+    value: 0
+    units: dimensionless
+  salaries_percent_FCI:
+    value: 0
+    units: 1/year
+  benefit_percent_of_salary:
+    value: 0
+    units: dimensionless
+  maintenance_costs_percent_FCI:
+    value: 0
+    units: 1/year
```

Comment:
@tarnold17 should this be 0.02?

Reply:
Yes, good catch - addressed in latest commit.

FWIW, later when I go to write the report for this flowsheet, I will probably need to push some more changes to the display results methods. I am not too concerned about that immediately though. For now, I am mostly just trying to get the files off of Box and into the repo.